src/Magento/FunctionalTestingFramework/Util/Filesystem/RecursiveGlobUtil.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\FunctionalTestingFramework\Util\Filesystem;
+
+class RecursiveGlobUtil
```

Comment:
I think it's not necessary to create  `RecursiveGlobUtil`  you can make use of `Magento\Mtf\Config\FileResolver\Mask` and specify proper filename mask in `di.xml`

Reply:
Fixed